iron-flex-layout.html

```diff
@@ -58,6 +58,17 @@
 @pseudoElement iron-flex-layout
 @demo demo/index.html
 -->
+<script>
+  // TODO (emarquez): remove when we have a fleshed out /deep/ solution
+  // required for backwards compat for projects that rely on a global [hidden[
```

Comment:
I mean, there is no deep solution coming, so I would just make the comment something like:

>This is left only for backward compatibility with projects that incorrectly relied on unscoped global [hidden] rules; removing would be a breaking change, but new projects should never rely on this.

Also, second bracket on hidden is pointing the wrong way.